curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/fill-in-the-blank-with-placeholder-text.md

```diff
@@ -9,19 +9,19 @@ dashedName: fill-in-the-blank-with-placeholder-text
 
 # --description--
 
-Web developers traditionally use <dfn>lorem ipsum text</dfn> as placeholder text. The lorem ipsum text is randomly scraped from a famous passage by Cicero of Ancient Rome.
+Web developers traditionally use <dfn>lorem ipsum text</dfn> as placeholder text. The `lorem ipsum` text is randomly scraped from a famous passage by Cicero of Ancient Rome.
 
-Lorem ipsum text has been used as placeholder text by typesetters since the 16th century, and this tradition continues on the web.
+`Lorem ipsum` text has been used as placeholder text by typesetters since the 16th century, and this tradition continues on the web.
 
-Well, 5 centuries is long enough. Since we're building a CatPhotoApp, let's use something called "kitty ipsum text".
+Well, 5 centuries is long enough. Since we're building a CatPhotoApp, let's use something called "`kitty ipsum`" text.
```

Comment:
```suggestion
Well, 5 centuries is long enough. Since we're building a CatPhotoApp, let's use something called "kitty ipsum" text.
```
I'm not sure how scare quotes will translate, but it's definitely a question for the translators.